Whiskey/en-US/about_Whiskey_Tasks.help.txt

```diff
@@ -17,6 +17,8 @@ LONG DESCRIPTION
     * `DotNetPack`: runs `dotnet pack` command.
     * `Exec`: runs an executable/program.
     * `GetPowerShellModule`: installs a PowerShell module.
+    * `GitHubRelease`: creates a release in GitHub and uploads files for that 

```

Comment:
Missing `about_Whiskey_GitHubRelease_Task` documentation.

Reply:
I wrote one but forgot to add it. Fixed.